clj-refactor.el

```diff
@@ -2472,8 +2472,8 @@ possible choices. If the choice is trivial, return it."
 
 
 (defun cljr--add-project-dependency (artifact version)
-  (let* ((project-file (cljr--project-file))
-         (deps (cljr--project-with-deps-p project-file)))
+  (if-let ((project-file (cljr--project-file))
```

Comment:
Please also check that it ends on project.clj or deps.edn

(the other results of `(cljr--project-file)` are unsupported)